scripts/2.mutation-classifier.py

```diff
@@ -234,21 +289,26 @@ def select_feature_set_columns(X, feature_set):
     cv_results_df = cv_results_df.append(df)
 
 
-# In[17]:
+# In[18]:
+
 
 # Cross-validated performance heatmap
-cv_score_mat = pd.pivot_table(cv_results_df,
-                              values='mean_test_score', 
-                              index='feature_set',
-                              columns='classify__alpha')
-ax = sns.heatmap(cv_score_mat, annot=True, fmt='.1%')
-ax.set_xlabel('Regularization strength multiplier (alpha)')
-ax.set_ylabel('Feature Set');
+cv_results_df['classify__alpha'] = cv_results_df['classify__alpha'].astype(str)
+
+(ggplot(cv_results_df, aes(x='classify__alpha',
+                           y='mean_test_score',
+                           fill='feature_set'))
+ + geom_bar(stat='identity', position='dodge')
```

Comment:
I'm not sure a barplot is the best. I'm thinking a scatterplot with model shown via line/dot color or via facet.

You could do a [`scale_x_log10`](http://plotnine.readthedocs.io/en/stable/generated/plotnine.scales.scale_x_log10.html) so the x-axis is continuous rather than discreet.

Ideally plot would expand_limits for y to include 0.5 and 1. This will have to wait until the next plotnine release: see https://github.com/has2k1/plotnine/commit/192609d761726c84ffec38dcbade325d9556364e. In other words, 0.5-1.0 should be the ylim unless AUROCs are below 0.5 in which case the lower bound should accommodate the lowest AUROC.